app/services/albumService.js

```diff
@@ -0,0 +1,31 @@
+const fetch = require('node-fetch');
+
+const config = require('../../config').common.albums;
+
+module.exports = {
+  getAlbums: async () => {
```

Comment:
For best practices we can export the functions of services directly as I commented more above.